ld_script.ld

```diff
@@ -3,35 +3,36 @@ ENTRY(Start)
 gNumMusicPlayers = 4;
 gMaxLines = 0;
 
+MEMORY
+{
+    EWRAM (rwx) : ORIGIN = 0x2000000, LENGTH = 256K
+    IWRAM (rwx) : ORIGIN = 0x3000000, LENGTH = 32K
+    ROM    (rx) : ORIGIN = 0x8000000, LENGTH = 16M
```

Comment:
`ROM` size should be `32`, not `16`.

Reply:
Hi sorry for the delayed response.
Isn't the base cartridge 16mb or does it auto-expand when exceeding that limit?

Reply:
Correct, there's otherwise no limit here. `gbafix` will just pad the ROM size up